packages/tools/client-logger/app-insights-logger/src/packageVersion.ts

```diff
@@ -0,0 +1,9 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ *
+ * THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY
+ */
+
+export const pkgName = "@fluid-tools/app-insights-logger";
```

Comment:
these are not used, so no need to generate this file.

Reply:
Removed in commit 70aa9f